src/screens/Ticketing/Purchase/Overview/index.tsx

```diff
@@ -39,15 +40,18 @@ const PurchaseOverview: React.FC<OverviewProps> = ({
     params.toTariffZone,
   );
 
-  const [selectedPreassignedFareProduct, setSelectedPreassignedFareProduct] =
-    useState(preassignedFareProduct);
+  const setSelectedPreassignedFareProduct = (fp: PreassignedFareProduct) => {
```

Comment:
Maybe rename to onSelectPreassignedFareProduct?

Reply:
✅